tests/modules/nf-core/plink/recode/main.nf

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { PLINK_RECODE } from '../../../../../modules/nf-core/plink/recode/main.nf'
+include { PLINK_VCF } from '../../../../../modules/nf-core/plink/vcf/main.nf'
+
+workflow test_plink_recode {
+
+    input = [
+        [ id:'test', single_end:false ], // meta map
+        file(params.test_data['homo_sapiens']['genome']['syntheticvcf_short_vcf_gz'], checkIfExists: true)
+    ]
+    PLINK_VCF ( input )
+
+    bed_ch = PLINK_VCF.out.bed
```

Comment:
Don't forget to update the tests!

Reply:
Modified the test so that it now works with 3 inputs: https://github.com/nf-core/modules/pull/2683/commits/6e6d4c666a8b4ca60f719bd25a373e09b87c69de